TeaDiary/Scripts/App/constants.js

```diff
@@ -0,0 +1,13 @@
+﻿var ApiCommands = new apiCommands();
+var ApiQueries = new apiQueries();
+
+var failureText = "Fail";
+var successText = "Success";
+
+var searchType = {
+    Name: "name",
+    Type: "type" 
+};
+
+var teaListTitle = $("<h2>").html("Your tea list:");
+var noTeasTitle = $("<h2>").html("No teas were found :(");
```

Comment:
I'd rather you have a `"No teas were found :("` constant and do `$("<h2>").html(` on the go